src/tools/ci/install_meson_and_rebase.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/sh
+
+set -e
+set -x
+
+case $CIRRUS_OS in
+    freebsd | linux | darwin | windows | mingw)
+    ;;
+    *)
+        echo "unsupported operating system ${CIRRUS_OS}"
+        exit 1
+    ;;
+esac
+
+# After repartition, hidden files are not copied. Copy them to
+# working dir
+if [ "$CIRRUS_OS" = 'freebsd' ]; then
+    cp -r $CIRRUS_WORKING_DIR.orig/.[^.]* $CIRRUS_WORKING_DIR/
+    chown -R postgres:postgres .[^.]*
+fi
+
+
+# install meson by using pip
+install_meson () {
+    case $CIRRUS_OS in
+        windows)
+            PYTHON=python
+        ;;
+        *)
+            PYTHON=python3
```

Comment:
I assume there's no 'python3' on windows? If not, perhaps we should just add an alias to the image? That's annoying in other places too...